src/commands.js

```diff
@@ -17,6 +17,10 @@ module.exports = new Map([
     footer: { text: 'uwu' }
   }) ],
 
+  [ toad('invite', 'come sing for me'), async ({ client }) =>({
+    description: `click [here](https://discord.com/oauth2/authorize?client_id=${client.application.id}&permissions=0&scope=bot 'la la la') and i'll come sing in your server too`
```

Comment:
`la la la`? won't that break the link?

Reply:
Nope thats just the hover text. The link still works fine. I actually figured out how to do that by looking at the Frogs help page.

Reply:
oh right, forgot that was a thing